cypress/support/commands.js

```diff
@@ -22,4 +26,45 @@
 //
 //
 // -- This will overwrite an existing command --
-// Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
\ No newline at end of file
+// Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+
+const workerDate = {
+  firstName: faker.person.firstName(),
+  lastName: faker.person.lastName(),
+  email: faker.internet.email(),
+  age: Math.floor(Math.random() * (65 - 18 + 1)) + 18,
+  salary: Math.floor(Math.random() * (10000 - 100 + 1)) + 100,
+  department: faker.commerce.department()
+};
+
+Cypress.Commands.add('createWorker', () => {
+  cy.get('button[id="addNewRecordButton"]')
+    .click();
+  cy.get('input[id="firstName"]')
+    .type(workerDate.firstName);
+  cy.get('input[id="lastName"]')
+    .type(workerDate.lastName);
+  cy.get('input[id="userEmail"]')
+    .type(workerDate.email);
+  cy.get('input[id="age"]')
+    .type(workerDate.age);
+  cy.get('input[id="salary"]')
+    .type(workerDate.salary);
```

Comment:
You are trying to type a number into an input field using cy.type(). This won't work because cy.type() only works with strings. You need to convert the number to a string first.